spring-cloud-gcp-docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -33,15 +33,12 @@ It provides the common set of operations needed to interact with Google Cloud Pu
 https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-pubsub[Google Cloud Java Client for Pub/Sub]
 to interact with Google Cloud Pub/Sub.
 
-`PubSubTemplate` depends on a `PublisherFactory`, which is a functional interface to provide a
-Google Cloud Java Client for Pub/Sub `Publisher`.
-The Spring Boot starter for GCP Pub/Sub auto-configures a `PublisherFactory` with default settings
-and uses the `GcpProjectIdProvider` and `CredentialsProvider` auto-configured by the Spring Boot GCP
-starter.
+`PubSubTemplate` depends on a `PublisherFactory` and a `SubscriberFactory`.
+The `PublisherFactory` provides a Google Cloud Java Client for Pub/Sub `Publisher`.
+The `SubscriberFactory` provides the `Subscriber` for asynchronous message pulling, as well as a `SubscriberStub` for synchronous pulling and an `Acknowledger`, to automatically acknowledge incoming messages if a user so desires.
```

Comment:
how does the `Acknowledger` automatically ack incoming messages? Isn't the idea that you have to invoke it explicitly?

Reply:
I guess the part where you say "if a user so desires" is confusing me.

Reply:
I think I would just remove the part of the sentence: ", to automatically acknowledge incoming messages if a user so desires.".